saleor/graphql/product/mutations/products.py

```diff
@@ -444,6 +444,14 @@ class Meta:
         description = 'Creates a product image.'
         model = models.ProductImage
 
+    @classmethod
+    def clean_input(cls, info, instance, input, errors):
+        cleaned_input = super().clean_input(info, instance, input, errors)
+        uploaded_image = cleaned_input['image']
+        if not uploaded_image.content_type.startswith('image'):
```

Comment:
The correct mime type is `image/...` (https://www.iana.org/assignments/media-types/media-types.xhtml#image), you should check it's starting with `image/`.